app/view_models/assignment_repo_view.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
Let's start organizing these a little more what do you think about a path like

`/app/view_models/assignment_repo/show_view.rb`

```ruby
# frozen_string_literal: true

class AssignmentRepo::ShowView < SharedAssignmentRepoView
  # ...
end
```